src/components/TodoComponents/TodoForm.js

```diff
@@ -4,15 +4,17 @@ import { FaPlus } from 'react-icons/fa';
 const TodoForm = ({ inputHandler, addHandler, enterKeyHandler, text }) => {
```

Comment:
Noble great job on the assingment. Not only You completed the TodoApp but you added more style and functionality to the app and made it responsivness !
- Your code is clean and You are properly splitting up the code into multiple components. You are correctly using state and props in your project.
- Your git workflow is fluid and You did amazing job on this assingment 👍